libs/common-types/src/tokens.rs

```diff
@@ -22,61 +12,29 @@ pub enum PermissionedCurrency {
 pub enum CurrencyId {
 	// The Native token, representing AIR in Altair and CFG in Centrifuge.
 	Native,
+
 	// A Tranche token
-	Tranche(u64, [u8; 16]),
+	Tranche(PoolId, TrancheId),
 
 	/// Karura KSM
 	KSM,
```

Comment:
I assume you are referring to the natively handled tokens here, so `Native`?, `Tranche`, `KSM`, and `KUSD`/`AUSD`?

There is no real need to "migrate" them, i.e., drop these variants and just have them registered in the asset-registry. Specially not for the `Native` and `Tranche` tokens since those are really vital to have at the code level.

The other ones _can_ be migrated. My take on this is that there really isn't a clear advantage on dedicating time to doing that _now_, since that would involve migrating balances that are now stored under, say, `CurrencyId::KSM` to `CurrencyId::ForeignAsset(x)`. Acala, for example, has kept most of the tokens they initially handled natively.

The one case where we do need to run a migration for is for merging `kUSD` and `aUSD`. We can choose to do that now now but I certainly didn't want to do yet another thing in this PR :)

Let me know what you think